### DISCUSSION TITLE
Character position problem with respect to the rectangle

### DISCUSSION THREAD

**Discussion by Daimones81** [google/model-viewer#4291] [Q&A] [answered]
Hi I just started using Model-viewer and it's great. I have encountered only one problem at the moment and I don't know how to fix it. It happens to me only with a couple of super Mario bros models taken from sketchfab, while with other models always taken from sketchfab they work great. The problem is that when I view in ar the rectangle sits well on the floor, but the model displays very tall. His shadow is placed well on the floor but the character is not. And I have no way to get it down, because the (placement) rectangle is already on the ground. How can I change this problem? Thanks in advance.
It should be noted that before viewing in ar mode, then viewing on the browser, it works well and can be seen very well. The character is on the ground obviously attached to his shadow. The problem only happens when I view in ar mode

**Comment by Daimones81** [comment]
Good evening, can someone please help me? It's my 5 year old son's birthday in two days and I wanted to surprise him. But this very character with this animation, has this problem.

https://sketchfab.com/3d-models/star-dance-18c123dd557749b982baf0a2103f728d

**Reply by elalish** [reply]
It sounds pretty likely that the units are wrong, since AR is the only place where physical scale becomes apparent. See if you can fix it with `scale="0.01 0.01 0.01"` or similar: https://modelviewer.dev/docs/#entrydocs-scenegraph-attributes-scale

**Reply by Daimones81** [reply]
Hello and thanks for the answer. Where should I enter this? In the html page or in the css file If in the html page, where exactly should I put it? Thank you very much

**Reply by elalish** [reply]
Did you follow the link to our docs? It's in the attributes section, so you put it into the `<model-viewer>` element declaration. You can also set it via JS: `mv.scale=...`.

**Reply by Daimones81** [reply]
Yes, I'm following it but to no avail. Can you kindly help me. I tried both by entering 
After model-viewer 
scale="0.01 0.01 0.01" And it doesn't work I also created the <script> 
const updateScale = () => { modelViewerTransform.scale = `${x.value} ${y.value} ${z.value}`; }; </script> 
And called back UpdateScale() And it doesn't go the same

**Reply by elalish** [reply]
I can't really help without a repro link. There are other possible model problems too - sometimes they put a weird transform on the animation so it's way different than the resting pose; in that case you may need to fix it in Blender.

**Comment by Daimones81** [comment]
The link to the exact model, I sent above It's on sketchfab You have the link in my second post

**Reply by elalish** [reply]
The model is not a repro - I mean your website.

**Comment by Daimones81** [comment]
https://ilparadisodelpc.altervista.org/mario/index.html

**Comment by Daimones81** [comment]
The thing that drives me crazy is that all other models work. But I like this for the animation and texture HD. Maybe one thing I've noticed is that the problem might be with the animations done with Blender. Because other Mario models with jump animations but taken from mixamo work well. But the character has a bad texture .. LoL

**Comment by Daimones81** [comment]
Hi, I sent you the. Link to the site, but it didn't let me know anything about it. Unfortunately I'm having this problem even with other characters taken from sketchfab. Triangle (base) touches the ground well, the shadow is the same, but the character is taller. It is clear that it does not touch the ground. Plus as I told you, I can't even scale it, even after the <model-viewer scale="0.01 0.01 0.01"........ >

**Reply by elalish** [reply]
You're link doesn't work here - the GLB fails to load. Try https://modelviewer.dev/examples/scenegraph/#transforms to see how scale works in 3D and AR modes.

**Reply by Daimones81** [reply]
My link works great, you just have to wait a bit for the Mario Bross character to load.

https://ilparadisodelpc.altervista.org/mario/index.html

**Comment by Daimones81** [comment]
Good evening, so two important things. 
1) I managed to lower the character. I opened it in blender, it had the root bone detached from the body. I modified it and it worked. Now the character touches the ground. 
2) I also managed to scale, but it only works in the Viewer browser, when I open it in ar it returns to the original size. Why? Maybe I need to change the scale to ar too? How? Thanks for your help.

**Reply by elalish** [reply, marked as answer]
That'll only work in WebXR mode - to make it work in Quick Look as such you'll need to scale it in blender.
